public/app/plugins/datasource/mssql/partials/config.html

```diff
@@ -13,9 +13,9 @@ <h3 class="page-heading">MS SQL connection</h3>
 	</div>
 
 	<div class="gf-form">
-		<label class="gf-form-label width-7">Authentication</label>
```

Comment:
I would probably use this the other way around. Adding the id to the select and using a for attribute here.